test/src/index/transport/testAppiumOptions.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert');
+const Nightwatch = require('../../../lib/nightwatch.js');
+
+describe('Test Appium client', function () {
```

Comment:
we also need a negative test, to check if the new functions return `false` when they're supposed to.